python/beeai_framework/agents/experimental/remote/agent.py

```diff
@@ -190,3 +190,9 @@ def memory(self) -> BaseMemory:
     @memory.setter
     def memory(self, memory: BaseMemory) -> None:
         raise NotImplementedError()
+
+    async def clone(self) -> "RemoteAgent":
+        cloned = RemoteAgent(self.input.agent_name, url=self.input.url)
+        cloned.emitter = await self.emitter.clone()
+        cloned._is_running = self._is_running
```

Comment:
This should always be set to `False`; otherwise, you can't use the agent. Remove the whole line, please.

Reply:
https://github.com/i-am-bee/beeai-framework/pull/705/commits/b2e1541de8e70648d7be72b23d16b7216ee532f0